### PR TITLE
Open nightly feature gate for crossbeam-channel.

### PR DESCRIPTION
**Pull request by hicqu**
crossbeam-channel uses some new rust features behind nightly gate.
Considering TiKV also uses rust-nightly, let's open the gate.

### CODE REVIEW

**Comment by hicqu** [comment]
@UncP PTAL.

File: Cargo.toml [hunk 1]
```diff
@@ -105,6 +104,10 @@ optional = true
 version = "0.2"
 features = ["secure"]
 
+[dependencies.crossbeam-channel]
```
Comment (BusyJay) [inline Cargo.toml]: Why not use the same style as uuid?

Comment (siddontang) [reply Cargo.toml]: above grpcio also uses the same style.

Comment (BusyJay) [reply Cargo.toml]: So it can be changed too.

**Pull request merged**